src/main/java/Main.java

```diff
@@ -100,23 +103,53 @@ public static int CrudOption() {
 
     }
 
-    public static void CreatePatient(){
+    public static void createPatient(PatientClass patient,String DatabaseName){
+        PatientClass pat = new PatientClass();
+        System.out.println("Enter ID: ");
+        int ID = myInput.nextInt();
+        System.out.println("Enter Name: ");
+        String name = myInput.next();
+        System.out.println("Enter DOB: ");
+        String dateOfBirthday = myInput.next();
+        System.out.println("Enter Treatment Date: ");
+        String dateOfTreatment= myInput.next();
+        System.out.println("Enter Address: ");
+        String address = myInput.next();
+        System.out.println("Enter Age: ");
+        int age = myInput.nextInt();
+        System.out.println("Enter Allergies: ");
+        String allergies = myInput.next();
+        System.out.println("Is Special Needs?: ");
+        boolean needspecialNeeds = myInput.nextBoolean();
+        System.out.println("Enter Treatment Type: ");
+        String typeOfTreatment = myInput.next();
+        pat.setID(ID);
+        pat.setName(name);
+        pat.setDateOfBirthday(dateOfBirthday);
+        pat.setDateOfTreatment(dateOfTreatment);
+        pat.setAddress(address);
+        pat.setAge(age);
+        pat.setNeedspecialNeeds(needspecialNeeds);
+        pat.setTypeOfTreatment(typeOfTreatment);
+        implement.createPatient(pat);
```

Comment:
```suggestion
        implement.createPatient(pat,// Write the database name over here );
```
Read the Crud class and look at what the class need and whatever it need send it in 
the createPatient need a Patient object AND the database name